src/audio.c

```diff
@@ -844,6 +844,9 @@ static int aurx_stream_decode(struct aurx *rx, const struct rtp_header *hdr,
 
 		rx->ssrc = hdr->ssrc;
 	}
+	else if (hdr->m) {
+		aubuf_flush(rx->aubuf);
+	}
```

Comment:
This is not needed. The aubuf runs empty --> "underrun", then the `ajb` object can be reset. A `rem` PR will follow.

Reply:
- In case of an underrun there is a fix in `rem`: https://github.com/baresip/rem/pull/51
- In case of SSRC change, there is already a `aubuf_flush`
- In case of timestamp jump: Needs a fix here in `audio.c`, maybe in combination with marker bit.